### ISSUE TITLE
[Bug] Switch buttons don't have accessible name

### ISSUE THREAD

**Issue by satelllte** [satelllte/displacementx#111]
**Describe the bug**

The switch buttons don't have accessible names:

```html
<button type="button" role="switch" aria-checked="true" data-state="checked" value="on" ...
```

Report:
https://pagespeed.web.dev/analysis/https-displacementx-pages-dev/pfrkfl33b1?form_factor=desktop

**Expected behavior**

The switch buttons should have accessible names and shouldn't fail on accessibility report
